demisto_sdk/scripts/validate_file_permission_changes.py

```diff
@@ -11,106 +10,61 @@
-    "Invalid value for CI env var. Expected 'true' or 'false', actual '{env_var_str}'"
-)
-ERROR_INPUT_FILES_INVALID = "Invalid value for input files: {input}"
-CI_ENV_VAR = "CI"
-
-
-def split_files(files_str: Union[str, List[str]]) -> List[str]:
-    """
-    Helper function to return a `list` of `str`
-    from an input string.
-
-    Args:
-    - `files_str` (``str | List[str]``): The input files.
-
-    Returns
-    - `List[str]` containing a list of strings.
-    """
-
-    if isinstance(files_str, list):
-        return files_str
-    elif isinstance(files_str, str):
-        return files_str.split() if files_str else []
-    else:
-        raise typer.BadParameter(ERROR_INPUT_FILES_INVALID.format(input=files_str))
-
-
-def is_ci() -> bool:
-    """
-    Helper function to detect whether we're running
-    in a CI environment. To detect this, we rely on the `CI` env var.
-
-    Returns:
-    - `True` if we're in a CI environment, `False` otherwise.
-    """
-
-    if os.getenv(CI_ENV_VAR):
-        env_var_str = os.getenv(CI_ENV_VAR, "false").lower()
-        if env_var_str in {"true", "1", "yes"}:
-            return True
-        elif env_var_str in {"false", "0", "no"}:
-            return False
-        else:
-            raise typer.BadParameter(
-                ERROR_IS_CI_INVALID.format(env_var_str=env_var_str)
-            )
-    else:
-        return False
-
 
 @main.command(help="Validate that file modes were not changed")
 def validate_changed_files_permissions(
-    changed_files: List[str] = typer.Argument(
-        default=None, help=HELP_CHANGED_FILES, callback=split_files
-    )
+    changed_files: List[Path] = typer.Argument(
+        default=None,
+        help="The files to check, e.g. dir/f1 f2 f3.py",
+        exists=True,
+        file_okay=True,
+        dir_okay=False,
+    ),
+    ci: bool = typer.Option(envvar="CI", default=False),
 ) -> None:
     """
     Validate whether the file mode was modified. Exit code 0 if no files
     modes were modified, 1 otherwise.
 
     Args:
-    - `changed_files` (``List[str]``): The files to check, e.g. 'test/f1 f2 f3.py'.
+    - `changed_files` (``List[Path]``): The files to check, e.g. 'test/f1 f2 f3.py'.
+    - `ci` (``bool``): Whether we're in a CI environment or not.
     """
 
     exit_code = 0
 
     logging_setup()
     git_util = GitUtil.from_content_path()
 
-    ci = is_ci()
-
     logger.debug(f"Running in CI environment: {ci}")
 
-    if changed_files:
-        logger.debug(f"Got {','.join(changed_files)} as input...")
-    else:
+    logger.debug(f"Input {changed_files=}")
+
+    if not changed_files:
         logger.debug(
             f"Getting changed files from git branch '{DEMISTO_GIT_PRIMARY_BRANCH}'..."
         )
+        git_util = GitUtil.from_content_path()
         changed_files = [
-            str(path)
-            for path in git_util.get_all_changed_files(DEMISTO_GIT_PRIMARY_BRANCH)
+            Path(git_util.git_path()) / Path(path)
```

Comment:
```suggestion
            Path(git_util.git_path(), path)
```

Reply:
Removed as part of `git` change file detection, https://github.com/demisto/demisto-sdk/pull/4405/commits/24b66fd5ba7781ebe5267ea8cc5004a29f9b21a1